monai/tests/test_clinical_preprocessing.py

```
@@ -0,0 +1,84 @@
+        preprocess_dicom_series("dummy_path.dcm", 123)
+
+
+# ------------------------
+# Tests for valid modalities
+# ------------------------
+
+@patch("monai.transforms.clinical_preprocessing.get_ct_preprocessing_pipeline")
+def test_preprocess_dicom_series_ct(mock_pipeline):
+    """Test preprocess_dicom_series successfully runs for CT modality."""
+    dummy_output = "ct_processed"
+    mock_pipeline.return_value = lambda x: dummy_output
```

Ruff (0.14.8)

Unused lambda argument: `x`

(ARG005)